docker-tests/parameters.yml

```diff
@@ -1,6 +1,9 @@
 parameters:
     app_name: docker
     encryption_key: 123456789012345678901234567890ab
+    kms_key_id: alias/docker-testing
+    kms_key_region: us-east-1
+    stack_id: %kms_key_region%
```

Comment:
? tohle bude chtít taky nějak poladit, ale až třeba v nový verzi docker runneru? :-)

Reply:
poladit co?

Reply:
aby hodnoty nebyly zahardcodovaný

Reply:
tohle je jen pro testy, jinak se berou parameters z S3, nekde to ID klice byt musi